README.md

```diff
@@ -105,7 +105,13 @@ would be represented as:
 The projection authority is some domain governing body that defines  projections. 
 The contents of this field are permissive and any string can be provided. In practice, it is preferable 
 to use a known authority. 
-Examples include `epsg`, `iau_2015`, and `ogc`.
+
+| Authority Name | proj:authority | URL |
+| -------------- | -------------- | --- |
+| COSMO          | `cosmo`          |     |
+| European Petroleum Survey Groups (EPSG) | `epsg` | http://www.opengis.net/def/crs/EPSG |
```

Comment:
The authoritive source for this is actually epsg.org

Reply:
Updated the link. Where is it documented whether epsg or ogc are the authoratitative source?

Reply:
Well, EPSG.org is the authority that publishes the database, OGC just mirrors it.
Source is epsg.org itself: "This site contains the master EPSG Dataset."